Documentation/bpf/progtypes.rst

```diff
@@ -263,7 +263,7 @@ cases.
 ..
 
 One example of XDP BPF production usage is Facebook's SHIV and Droplet
-infrastructure which implement their L4 load-balancing and DDoS countermeasures.
+infrastructure which implements their L4 load-balancing and DDoS countermeasures.
```

Comment:
I'd rather add an “s” to “infrastructure” instead, as [SHIV and Droplet](https://netdevconf.org/2.1/slides/apr6/zhou-netdev-xdp-2017.pdf) are two different things.